google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -149,6 +150,12 @@ MATCHER_P(HasReturnStats, return_commit_stats, "has return-stats value") {
   return arg.return_commit_stats() == return_commit_stats;
 }
 
+MATCHER_P(HasMaxCommitDelay, max_commit_delay, "has max commit delay") {
+  return arg.max_commit_delay() ==
+         google::protobuf::util::TimeUtil::MillisecondsToDuration(
+             max_commit_delay.count());
```

Comment:
Here is a point where we could well do with a `duration_cast<milliseconds>()`, so that we can pass any duration unit.

Reply:
Done.